plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -147,8 +146,7 @@ public ColumnFamilyHandle getSegmentIdentifierByName(final SegmentIdentifier seg
   }
 
   @Override
-  public Optional<byte[]> get(final ColumnFamilyHandle segment, final byte[] key)
-      throws StorageException {
+  public Optional<byte[]> get(final ColumnFamilyHandle segment, final byte[] key) {
```

Comment:
I know it's a runtime exception, but I think showing it in the signature makes it clear you should expect it.  I don't think we have a coding standard on this except for it must compile.